### ISSUE TITLE
index.md uses HTML formatting for no apparent reason

### ISSUE THREAD

**Issue by hlapp** [datacarpentry/lesson-template-ARCHIVED#12]
Specifically, for this block it switches to HTML formatting, even though there isn't anything apparent that can't just as well be formatted in Markdown, consistent with the rest of the document.

``` html
<br>

<h2>Requirements</h2>

<p>
Data Carpentry's teaching is hands-on, so participants are encouraged to use
their own computers to insure the proper setup of tools for an efficient workflow.
<em>These lessons assume no prior knowledge of the skills or tools</em>, but working 
through this lesson requires working copies of the software described below.
To most effectively use these materials, please make sure to install everything 
<em>before</em> working through this lesson.
</p>
```

**Comment by tracykteal** [comment]
It's not no apparent reason. It's because I was involved and still think in HTML first. Mentally shifting to Markdown. :)

**Comment by hlapp** [comment]
> It's because I was involved

:smile:

**Comment by hlapp** [comment]
Fixed in #14.
